examples/python/sso/validate_jwt.py

```diff
@@ -15,70 +15,61 @@
 
 import requests
 from jose import jwt
-from jose.exceptions import ExpiredSignatureError, JWTError, JWTClaimsError
+from jose.exceptions import ExpiredSignatureError, JWTError
 
 
-def validate_eve_jwt(jwt_token):
+def validate_eve_jwt(jwt_token: str) -> dict:
     """Validate a JWT token retrieved from the EVE SSO.
 
     Args:
         jwt_token: A JWT token originating from the EVE SSO
-    Returns
-        dict: The contents of the validated JWT token if there are no
-              validation errors
+    Returns:
+        The contents of the validated JWT token if there are no validation errors
     """
-
-    jwk_set_url = "https://login.eveonline.com/oauth/jwks"
-
-    res = requests.get(jwk_set_url)
+    # fetch JWKs from endpoint
+    res = requests.get("https://login.eveonline.com/oauth/jwks")

```

Comment:
Instead of this URL being hardcoded, any metadata/endpoints should be looked up from the official metadata/description endpoint on the SSO at https://login.eveonline.com/.well-known/oauth-authorization-server

Reply:
Thank you for the feedback. I have included your change and also factored out all the other hard coded values into constants, along with some variable renaming to further improve readability.